docs/book/src/topics/azure-cni-v1.md

```diff
@@ -0,0 +1,138 @@
+# Using Azure CNI V1
```

Comment:
Want to move this to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/docs/book/src/topics/addons.md#cni instead of its own page?

Reply:
Thanks for pointing it out. This doc should be part of addons. Moving it.